README.md

```diff
@@ -1,10 +1,28 @@
 Data from DECam to validate the performance of the LSST DM stack.
-Data from COSMOS field (Dey, PropID: 2013A-0351)
+Data from COSMOS field (Dey, NOAO PropID: 2013A-0351)
 
 Data are used by `validate_drp`.
 
+Currently just includes calibrated image data with weightmaps and mask:
+Files
+------
+path      | description
+:---------|:-----------------------------
+`instcal` | Photometrically and astrometrically calibrated data
+          |   as processed by the NOAO DECam Community Pipeline
+`dqmask`  | Mask for `instcal` images
+`wtmap`   | Weightmap for `instcal` images
+
+No astrometric catalogs are included.
+
+Future directions include
+1. Add astrometric and photometric catalogs to re-check the `instcal` calibration
+2. Add raw versions of images + calibration data (bias, flat, dark) to enable full ISR processing of data.
+
+GIT-LFS
+------
 This repository uses LFS to store the large files outside of GitHub.
 
-For more details see:
+For more details about git-lfs and how it's used in LSST Data Management see:
 
 http://docs.lsst.codes/en/latest/development/git_lfs.html
```

Comment:
The link to git-lfs technotes seems to have changed and doesn't work. This works
http://docs.lsst.codes/en/latest/tools/git_lfs.html